election-api-typescript/README.md

```diff
@@ -0,0 +1,54 @@
+## Description
+
+If you are not familiar with how elections work in the UK, please see this short BBC video https://www.youtube.com/watch?v=cRxUhGetEPQ
+
+The results API presents a simple elections result service.
+
+### Domain
+
+The domain for the election represents some key concepts:
+
+- _**constituencyId**_ a unique integer id to identify a location. E.g "Brent Central" is 90
+- _**party**_ is a short 3, or 4, letter code for a party for instance LAB = Labour, CON = Conservative etc.
+- _**votes**_ the number of votes gained by a party in a constituency
+- _**share**_ the % share of the total votes the party received
+
+### API
+
+The API has 3 endpoints:
+
+- GET `/result/{id}` to get an elections result for a given id.
+- POST `/result` to add a new result
+- GET `/scoreboard` to get the running totals. This is unimplemented.
+
+## Setup
+
+Requires:
+
+- node (18) - available from https://nodejs.org/en/download/
+- npm - will be installed with node, or see https://docs.npmjs.com/downloading-and-installing-node-js-and-npm
+
+```
+npm install
+npm start
```

Comment:
As it stands wouldn't this need to be
```suggestion
npm install
npm build
npm start
```
?

What I think might be better is to install and use [`ts-node-dev`](https://www.npmjs.com/package/ts-node-dev) and replace the `start` command in the `package.json` file with:

`ts-node-dev --respawn --transpile-only ./src/index.ts`

Which removes the need for the build step and keeps the setup instructions the same between the JS version and this version.